packages/stellar_solana_bridge/src/services/stellar.ts

```diff
@@ -0,0 +1,49 @@
+import { Asset, Keypair, Memo, Networks, Operation, TransactionBuilder } from "@stellar/stellar-sdk";
+import * as StellarSdk from "@stellar/stellar-sdk";
+import bs58 from "bs58";
+import { Buffer } from "buffer";
+
+const STELLAR_HORIZON = "https://horizon.stellar.org";
+
+const server = new StellarSdk.Horizon.Server(STELLAR_HORIZON);
+// ThreeFold Bridge Details
+const BRIDGE_ADDRESS = "GAC65XJVXMBPHKGVWRR4ZX6MA66I4JAOEAIMOVRIXALL3NO67UYMIWMZ";
+const TFT_ASSET = new Asset("TFT", "GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47");
+
+export const transferTFT = async (senderSecret: string, solanaRecipientAddress: string, amount: string) => {
+  const senderKeypair = Keypair.fromSecret(senderSecret);
+  const senderAccount = await server.loadAccount(senderKeypair.publicKey());
+
+  // Convert Solana address to 32-byte memo
+  const solanaAddressBytes = bs58.decode(solanaRecipientAddress);
+  if (solanaAddressBytes.length !== 32) {
+    throw new Error("Invalid Solana address length");
```

Comment:
please add this validation in the text-field toaddress rules so user does not have to wait after submitting form to get a validation err

Reply:
I prefer that too, you can decode it and make sure the length in bytes is 32